tests/foreman/ui/test_subscription.py

```diff
@@ -554,3 +556,79 @@ def test_positive_prepare_for_sca_only_subscription(target_sat, function_entitle
+
+    :id: 29ab0e80-25eb-44f0-9294-3a26922c33c5
+
+    :setup:
+        1. create new organization
+
+    :steps:
+        1. Upload expired manifest in newly created org
+        2. go to Content > Subscriptions page, click on 'Manage Manifest' button.
+        3. Search for message string 'Manifest expired', or 'Your manifest expired'
+        4. delete expired manifest from this org (cleanup part)
+        5. Upload non-expired manifest
+        6. go to Content > Subscription page, click on 'Manage Manifest' button.
+        7. Search for message string 'Manifest expiring soon', or 'Your manifest will expire'
+        8. delete expired manifest from this org (cleanup part)
+
+    :expectedresults:
+        1. 'Manifest expired', 'Manifest expiring soon' messages appear on Manage Manifest modal box
+
+    :BZ: 2075163
+
+    :customerscenario: true
+    """
+    remote_path = f'/tmp/{EXPIRED_MANIFEST}'
+    target_sat.put(DataFile.EXPIRED_MANIFEST_FILE, remote_path)
+    # upload expired manifest
+    target_sat.cli.Subscription.upload({'organization-id': function_org.id, 'file': remote_path})
+
+    with target_sat.ui_session() as session:
+        # Message - Manifest expired
+        session.organization.select(function_org.name)
+        # read expire manifest message
+        expired_manifest = session.subscription.read_subscription_manifest_header_message_and_date()
+        assert 'Manifest expired' in expired_manifest['header'], 'Manifest expire alert not found'
+        assert 'Your manifest expired' in expired_manifest['message']
+        assert 'import a new manifest' in expired_manifest['message']
+        # Cleanup - delete expired manifest
+        session.subscription.delete_manifest(
+            ignore_error_messages=['Danger alert: Katello::Errors::UpstreamConsumerNotFound']
+        )
+
+        # Message - Manifest expiring soon
+        # Upload non-expire manifest
+        session.organization.select(default_org.name)
+        target_sat.upload_manifest(default_org.id, function_sca_manifest.content)
+        original_expire_date = (
+            session.subscription.read_subscription_manifest_expiration_date_only()
+        )
+        # Initially 'Manifest expiring soon' message not found
+        # then update the 'Expire soon days' value from settings > Content
+        # value should be greater than 365 to get expected output message
+        setting_update.value = 366
+        setting_update = setting_update.update({'value'})
+        session.browser.refresh()
+        # read expire manifest message
+        expiring_soon = session.subscription.read_subscription_manifest_header_message_and_date()
+        assert (
+            'Manifest expiring soon' in expiring_soon['header']
+        ), 'Manifest expire alert not found'
+        assert 'Your manifest will expire' in expiring_soon['message']
+        assert 'import a new manifest' in expiring_soon['message']
```

Comment:
Maybe we could check the date here too (similar to expired manifest), since we know it will expire in `datetime.now() + timedelta(years=1)`. Some date formatting with `strptime` or `strftime` will be needed.

Reply:
I am not sure about the expire date here in this message, as we are refreshing page before calling `expiring_soon = session.subscription.read_subscription_manifest_header_message_and_date()` after that date changes to creation date + 1 year